relayer/src/worker/packet.rs

```diff
@@ -267,17 +267,35 @@ fn acknowledgment_metrics(path: &Packet, summary: &RelaySummary) {
 #[cfg(feature = "telemetry")]
 fn timeout_metrics(path: &Packet, summary: &RelaySummary) {
     use ibc::events::IbcEvent::TimeoutPacket;
-    let count = summary
+    let mut count = 0;
+    summary
         .events
         .iter()
         .filter(|e| matches!(e, TimeoutPacket(_)))
```

Comment:
To make this slightly more readable.
```suggestion
        .filter(|event| matches!(event, TimeoutPacket(_)))
```